lerna.json

```diff
@@ -0,0 +1,17 @@
+{
+  "lerna": "2.0.0",
+  "version": "0.11.0",
+  "packages": [
+    "packages/*"
+  ],
+  "changelog": {
+    "repo": "shopify/slate",
+    "labels": {
+      "enhancement": ":rocket: Enhancement",
+      "bug": ":bug: Bug Fix",
+      "documentation": ":memo: Documentation",
+      "internal": ":house: Internal"
```

Comment:
Internal refers to behind the scenes changes that don't necessarily change anything for users. This PR is an example of an `internal` change.